tests/test_game.py

```diff
@@ -403,11 +409,81 @@ def test_get_routing_matrix_example_3():
     )
 
 
+def test_get_individual_entries_of_matrices_example():
+    """
+    Tests that the function returns a dask task and that the computed task
+    returns the expected tuple
+    """
+    task = get_individual_entries_of_matrices(
+        lambda_2=2,
+        lambda_1_1=2,
+        lambda_1_2=2,
+        mu_1=2,
+        mu_2=2,
+        num_of_servers_1=2,
+        num_of_servers_2=2,
+        threshold_1=2,
+        threshold_2=2,
+        system_capacity_1=4,
+        system_capacity_2=4,
+        buffer_capacity_1=2,
+        buffer_capacity_2=2,
+        alpha=0.5,
+        target=2,
+    )
+
+    assert da.is_dask_collection(task)
+    values = da.compute(task)
+    assert np.allclose(
+        values, ((2, 2, 0.5, -0.00046944342133137197, -0.00046944342133137197),)
+    )
+
+
+def test_compute_tasks():
+    """
+    Tests that dask tasks are computed as expected
+    """
+
+    @da.delayed
+    def inc(x):
+        return x + 1
+
+    @da.delayed
+    def double(x):
+        return x * 2
+
+    x = random.random()
+    y = random.random()
+
+    tasks = tuple((inc(x), double(y)))
+    assert compute_tasks(tasks, processes=None) == (x + 1, 2 * y)
```

Comment:
This is not a great test.

**IF** it failed for some specific value of `x` or `y` you would not be able to reproduce the failure. This is what `hypothesis` is for: it cleverly keeps track of failures and lets you reproduce them.

Reply:
> Re [c9bf46c](https://github.com/11michalis11/AmbulanceDecisionGame/commit/c9bf46c69c8e011ba6bfd84af03b345c47f3fa92) why are you rounding at all?

@drvinceknight This is a change I made that I was not at all sure about. It's one of the things I wanted to discuss in our meeting. Thinking about how I am planning to extract these data, I was just thinking that including 1 decimal (instead of 2) would just simplify things a bit (and wouldn't affect the problem much).

That being said I'm happy to change it back.

Reply:
You are absolutely right. I totally missed that